src/main/java/view/Play/YearSelectionPanel.java

```diff
@@ -0,0 +1,25 @@
+package view.Play;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class YearSelectionPanel extends JPanel {
+    private final JComboBox<String> yearComboBox = new JComboBox<>();
```

Comment:
Ensure that all added GUI components are styled consistently with other panels.

Reply:
Yup I added that